client/src/screens/patient-portal/PaymentMethods/PaymentMethods.js

```diff
@@ -80,7 +92,11 @@ const PaymentMethods = () => {
     fetchPaymentMethods();
   }, [fetchPaymentMethods]);
 
-  const onRowClick = (item) => {
+  const onItemEdit = (item) => {
+    setSelectedPayment(item);
+  };
+
+  const onItemDelete = (item) => {
     setSelectedPayment(item);
```

Comment:
We can merge `onItemEdit` and `onItemDelete` as `onItemEditOrDelete`.

Reply:
This is for the time being, once the API is ready we will use a different implementation.